### PR TITLE
[ros-hsrb_moveit*] new targets

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: ros-hsrc_description/install.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- type: target
+  name: hsr-setup
+
+- type: ros
+  source:
+    type: system
+    name: hsrc-description
```
Comment (PetervDooren) [inline ros-hsrc_description/install.yaml:7]: Why do we need the description of a robot we do not use?

Comment (MatthijsBurgh) [reply ros-hsrc_description/install.yaml:7]: Because one of the hsrb_moveit pkgs does depend on it. Naming and dependencies aren't always consistent.

**Comment by MatthijsBurgh** [comment]
@PetervDooren Be free to rename the branch or merge it into another branch

**Comment by MatthijsBurgh** [comment]
I think this can be merged independently.

**Comment by ar13pit** [comment]
@MatthijsBurgh Do you want me to have a look into this PR?

**Comment by MatthijsBurgh** [comment]
@ar13pit would be nice

File: ros-hsrc_description/install.yaml (hunk 1 above)
Comment (ar13pit) [inline ros-hsrc_description/install.yaml:7]: This target can be removed safely after removing it as a dependency in `hsrb_moveit_config` as it is just used in a launch file there.

Comment (ar13pit) [reply ros-hsrc_description/install.yaml:7]: Although if you want to keep this target for any later use, I'm okay with it

Comment (MatthijsBurgh) [reply ros-hsrc_description/install.yaml:7]: Keep it. I want to keep the changes to uostream limited.

**Pull request merged**